src/util/api/index.js

```diff
@@ -17,3 +17,15 @@ export const getOAuthLoginAPI = () => '/api/users/login';
 export const getOAuthCallbackAPI = () => `/api/users/login`;
 export const getPayPalPaymentAPI = () => `/api/civic/payment/paypal`;
 export const getLoginStatus = () => `/api/users/self`;
+
+export const getUserMinAPI = likerId =>
+  `https://${
+    IS_TESTNET ? 'api.rinkeby.like.co' : 'api.like.co'
+  }/api/users/id/${likerId}/min`;
+
+export const getLikeButtonTotalLikeCountAPI = (likerId, referrerURL) =>
+  `https://${
+    IS_TESTNET ? 'rinkeby.like.co' : 'like.co'
```

Comment:
this api is also accessible with `api.(rinkeby.)like.co`, so maybe make `api.(rinkeby.)like.co` a constant  `LIKE_CO_API_BASE`
also in `api.(rinkeby.).like.co` you can omit the `/api` part